src/Reducers/snackbarReducer.ts

```diff
@@ -0,0 +1,23 @@
+import { SnackbarAction, SnackbarState } from "../types/snackbar";
+
```

Comment:
You can put this in a global constant file or the folder root if the file is nested 

When a prop or function is used in more than one files this will make it more generic we cant put it here.
```suggestion
export const SEVERITIES = {
  SUCCESS: "success",
  INFO: "info",
  WARNING: "warning",
  ERROR: "error",
} as const;
```